client/src/App.js

```diff
@@ -34,6 +34,7 @@ function App() {
           <Route path="/" element={<Summary />} />
           <Route path="/register" element={<Register />} />
           <Route path="/login" element={<Login />} />
+          // I think you need to use a Navigate here instead of undefined for the not logged in case
```

Comment:
im pretty sure isLoggedIn returns false if the user is not logged in, so I'm not sure what you mean

Reply:
yes, but you don't want a route with `element={undefined}`
Which ill be the case if you have no user
`<Route path="/authorized" element={isLoggedIn && <Test />`
`{isLoggedIn && <Test />` returns `undefined` if isLoggedIn is falsy
You want a ternary with a Navigate as a fallback

`isLoggedIn ? <Test /> : <Navigate to="/dashboard" replace={true} />

Something like that